build-tests/api-extractor-scenarios/etc/apiItemKinds/api-extractor-scenarios.api.md

```diff
@@ -42,11 +30,27 @@ export interface IInterface {
 }
 
 // @public (undocumented)
-export namespace NamespaceContainingVariable {
-    let // (undocumented)
-    variable: object[];
-    let // (undocumented)
-    constVariable: object[];
+export namespace n1 {
+    // (undocumented)
+    export namespace n2 {
+        // (undocumented)
+        export class SomeClass3 {
+        }
+    }
+    // (undocumented)
+    export class SomeClass1 {
+    }
+    // (undocumented)
+    export class SomeClass2 extends SomeClass1 {
+    }
+        {};
+}
```

Comment:
This is also an instance of the bug mentioned in https://github.com/microsoft/rushstack/pull/3552#discussion_r930244544. Notice the empty `{}` and the fact that `SomeClass1` is exported here even though it's not actually exported in the original source file.

Reply:
`export { }`  is a special .d.ts directive that disables a deprecated semantics for modules. (I don't remember the details offhand, but I think it was something like, if NO declaration is exported, then ALL declarations are implicitly exported?) 

Here's [your test case in the playground](https://www.typescriptlang.org/play?#code/PQKhAIAEAcFcCMA2BLAxuEwBQBTAHtAPYBOALuAHYCGAtjgM7RWo6UCM4A3luOKolXr1wAZUJ0AwgKEdOAXx7h8RMn2nCxk9QCYleUjgoATDeJxTB9WQsXKS5anUbNWFXd1687q-pdFmLIQBmLgVeBRtQCBgEFHRMXAJ7SloGJhZ2Llskn3V-LUsAFlCsBSA) -- on the .d.ts tab, `export { }` appears inside `n1`.

`{};` without `export` is definitely a bug.